app/models/like.rb

```diff
@@ -1,4 +1,4 @@
 class Like < ApplicationRecord
-  belongs_to :author, class_name: 'User'
+  belongs_to :author, class_name: 'User', foreign_key: :author_id
```

Comment:
- [ ] Just like the previous reviewer suggested, please create a method that updates the likes counter. The resultant code will look something like this 👇 
``` ruby
def update_likes_counter
  post.increment!(:likes_counter)
end
```